docker-compose-dev.yml

```diff
@@ -17,6 +17,8 @@ services:
       SKIP_DEPLOY: "${SKIP_DEPLOY}"
       DATAVERSE_JSF_REFRESH_PERIOD: "1"
       DATAVERSE_FEATURE_API_BEARER_AUTH: "1"
+      DATAVERSE_MAIL_SYSTEM_EMAIL: "dataverse@localhost"
+      DATAVERSE_MAIL_MTA_HOST: "smtp"
```

Comment:
Should we add these to `docker/compose/demo/compose.yml` as well? Or defer this until later?

Reply:
Excellent idea!

Reply:
Done